templates/caffe-1.0.x/requirements.txt

```diff
@@ -0,0 +1 @@
+algorithmia>=1.0.0,<2.0
```

Comment:
Does it make sense to add pycaffe/caffe here, even though it's not installable directly via pypi?